ios/FluentUI/Separator/Separator.swift

```diff
@@ -49,12 +21,12 @@ open class Separator: UIView {
 
     @objc public override init(frame: CGRect) {
         super.init(frame: frame)
-        initialize(style: .default, orientation: .horizontal)
```

Comment:
Is it by design that we only have the one style?

Reply:
Yep!

Reply:
Good thing to call out in the description, then, and mark a note in our list of significant changes!